addons/sale_timesheet/models/project.py

```diff
@@ -320,6 +363,33 @@ def get_profitability_common(self):
             'timesheet_percentage_billable': timesheet_billable and timesheet_billable / (timesheet_billable + timesheet_non_billable) * 100 or 0.0,
         }
 
+    def _get_stat_buttons(self):
+        buttons = super(Project, self)._get_stat_buttons()
+        buttons.append({
+            'icon': 'clock-o',
+            'text': 'Billable Time',
+            'number': '%s %%' % (self.billable_time),
+            'action_type': 'action',
+            'action': 'hr_timesheet.act_hr_timesheet_line_by_project',
+            'additional_context': json.dumps({
```

Comment:
Why do you have to call json.dumps ?

Reply:
I had the same question 🙂 
I think it is to correctly format the dict in JSON. For instance, the key should have double quotes instead of single ones.